pkg/v1/tarball/image.go

```diff
@@ -26,8 +26,10 @@ import (
 	"path"
 	"path/filepath"
 	"sync"
+	"unsafe"
```

Comment:
Is there a lib to to track the size of all data that is referenced by this struct? `unsafe.Sizeof` returns the value 40 regardless of how many megabytes are already stored in `content` and `headers` map.
```go
type TarBuffered struct {
	tf      *tar.Reader
	pos     int
	headers map[int]*tar.Header
	content map[int][]byte
	EOF     bool
}
```